README.md

```diff
@@ -123,4 +123,5 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 | Brandon Varner       | Alabama's Cahaba River is known as "America's Amazon" |
 | Chinemelum Iloegbunam| I've been skydiving multiple times and in the proccess to get my license|
 | Kurt Mauritz         | The Earth is always flat on a map. |
-| Dylan Sarikas        | The Fun Fact about "eating 8 spiders in your sleep during your lifetime" is actually a myth
\ No newline at end of file
+| Dylan Sarikas        | The Fun Fact about "eating 8 spiders in your sleep during your lifetime" is actually a myth|
+|Alonzo Rice           | I went to a real estate class over the week-end | 
```

Comment:
Might want to add a space after `|` and before `A` for consistency, but not a major issue